manager/container.go

```diff
@@ -48,8 +48,12 @@ var enableLoadReader = flag.Bool("enable_load_reader", false, "Whether to enable
 var HousekeepingInterval = flag.Duration("housekeeping_interval", 1*time.Second, "Interval between container housekeepings")
 
 // cgroup type chosen to fetch the cgroup path of a process.
-// Memory has been chosen, as it is one of the default cgroups that is enabled for most containers.
-var cgroupPathRegExp = regexp.MustCompile(`memory[^:]*:(.*?)[,;$]`)
+// Memory has been chosen, as it is one of the default cgroups that is enabled for most containers...
+var cgroupMemoryPathRegExp = regexp.MustCompile(`memory[^:]*:(.*?)[,;$]`)
```

Comment:
random comment, out of scope for this PR: I wonder if this regex is really needed here. The format seems pretty easily parsable, 

e.g. from my system

```
11:pids:/system.slice/ssh.service,10:devices:/system.slice/ssh.service,4:cpu,cpuacct:/system.slice/ssh.service,2:memory:/system.slice/ssh.service,1:name=systemd:/system.slice/ssh.service,0::/system.slice/ssh.service
```

I think you could just split the string by commas and remove the leading `hierarchy-ID`, e.g. `11:`. We can also see which cgroup controller mounted by reading `/proc/cgroups`. Regex is totally fine for now (and what is used currently) but maybe worth a TODO as future improvement :)